packages/pretty-format/src/index.ts

```diff
@@ -506,6 +507,10 @@ const getConfig = (options?: OptionsReceived): Config => ({
     options && options.maxDepth !== undefined
       ? options.maxDepth
       : DEFAULT_OPTIONS.maxDepth,
+  maxWidth:
+    options && options.maxWidth !== undefined
+      ? options.maxWidth
+      : DEFAULT_OPTIONS.maxWidth,
```

Comment:
```suggestion
    options?.maxWidth ?? DEFAULT_OPTIONS.maxWidth,
```

probably same refactor for the others 🙂

Reply:
Sounds good, pushed one commit that contains the refactors together